src/radical/pilot/pilot_description.py

```diff
@@ -100,17 +101,18 @@ class PilotDescription(ru.TypedDict):
 
     .. data:: nodes
 
-       [type: `int` | default: `0`] [**NOT IN USE**] The number of nodes the
-       pilot should allocate on the target resource. This parameter is optional
-       and could be set instead of `cores` and `gpus` (and `memory`).
+       [type: `int` | default: `0`] The number of nodes the pilot should
+       allocate on the target resource. This parameter could be set instead of
+       `cores` and `gpus` (and `memory`).
 
-       .. note:: Either `cores` or `nodes` must be specified.  If `nodes` are
-                 specified, `gpus` must not be specified.
+       .. note:: Either `cores` or `nodes` **must** be specified.  If `nodes`
+                 are specified, `gpus` must not be specified.
```

Comment:
Will this create an issue? if the user ended in specifying both `gpus` and `nodes/cores`?

Reply:
Specifying `nodes` and `gpus` will raise an error.